### ISSUE TITLE
Likes and comments are not propagated to rest of Fediverse from pixelfed.de and other providers

### ISSUE THREAD

**Issue by ballerburg9005** [pixelfed/pixelfed#3385]
I am sorry if this is a duplicate. 

I have registered several accounts on Pixelfed instances to test this. Mostly pixelfed.de and pixelfed.social. 

I have written comments and given likes from those two very specifically to Mastodon instances and between the two. It seems that nothing comes through. The Pixelfed instances seem to be totally isolated from each other and I have given it plenty of time.

Is this a "normal" thing to expect, due to some known difficulties or bugs? 

It seems very bad and defeating to users who want to interact with each others posts. Only to notice after weeks that they don't get responses and replies to what they write, because no one can see it.

**Comment by kaibae19** [comment]
I have been experiencing this as well, I wasn't clear on if this was expected to work at this time.  Since I haven't found any documentation indicating such, I'll tag on here.  I am experiencing this as well.

Additionally, I have been able to send a direct message between Mastodon and Pixelfed, Mastodon didn't register that as a message but a post - also not sure if that was expected behavior in Mastodon.  But setting up two Pixelfed instances and following a user between the two, I am unable to see any posts for the remote system, but I do see details of the profile.  Again, I wasn't sure if this had been implemented yet or if this was an error.

**Comment by albertoventurini** [comment]
Same here. I set up my own instance of Pixelfed and followed some profiles from pixelfed.social. When they publish a photo, I can see it on my instance. If I like it, the like is successfully propagated to the remote instance (pixelfed.social). But if someone else likes (or comments) the photo on pixelfed.social, I don't see that activity on my instance.

I have observed the same behaviour between other instances, e.g. between pixelfed.social and photog.social.

It seems like the publisher of the photo receives all the likes and comments (either local or remote), but every other participant who acted on the photo won't get the entire interaction.

I would also be curious to know if this is a limitation of ActivityPub or something that has not been implemented in Pixelfed (yet).
